examples/time_handler_copy.py

```diff
@@ -0,0 +1,242 @@
+from builtins import object
```

Comment:
I would rather have time handler added to `sims_utils` as a general purpose utility class than have it copied here. We will also need it on other packages, besides `sims_ocs`. 

We could have it on a stand alone package as well but since most packages already depends on `sims_utils`, adding it there would make things easier I guess.

Reply:
Added to sims_utils : currently in branch feature/addTimeHandler.